docs/podman-image-trust.1.md

```diff
@@ -14,12 +14,12 @@ podman\-trust - Manage container registry image trust policy
 REGISTRY[/REPOSITORY]
 
 # DESCRIPTION
-Manages the trust policy of the host system. Trust policy describes
-a registry scope (registry and/or repository) that must be signed by public keys. Trust
-is defined in **/etc/containers/policy.json**. Trust is enforced when a user attempts to pull
+Manages the trusted keys for a container image usually dependent on its location.
```

Comment:
We manage other things the keys.  We setup which registiry and which repository are trusted.  Talking about secrets keys is really selling this short.

Reply:
OK, I've tried to rework it, if my latest isn't on point, please suggest further verbiage.